### DISCUSSION TITLE
Add a save function to Data and HeteroData

### DISCUSSION THREAD

**Discussion by skyelves** [pyg-team/pytorch_geometric#4729] [Q&A] [answered]
Currently I am working on implementing GNN models. Sometimes after training a few epochs, I want to stop training, and save the model and the graph. I can save the model with `torch.save(model.state_dict(), PATH)`. However, I didn't find a way to for me to save the data like `torch.save`. Though I can save it in a csv file, it may be much better if there is a built-in function in `Data` class and `HeteroData` class such as `save(dir/filename)` .  Also a complementary function `load(dir/filename)`.

**Comment by rusty1s** [comment, marked as answer]
You can also save the data via `torch.save(data, ...)` and load it via `torch.load`. Note that this is already automatically handled when using our `Dataset` and `InMemoryDataset` abstractions.

**Reply by skyelves** [reply]
Yeah it works, thanks for answering!
